### PR TITLE
Int 114 sdk get media

### PR DESCRIPTION
**Pull request by jfcsantos**
https://bynder.atlassian.net/browse/INT-117

### CODE REVIEW

File: includes/MediaBynderBrowser.inc [hunk 1]
```diff
@@ -9,22 +9,25 @@
...
-    $params = $this->params;
-    $build['form'] = drupal_get_form('media_bynder_add', $params);
+    /**
+     * Implements MediaBrowserPlugin::view().
+     */
+    public function view()
+    {
+        $build = array();
+        $params = $this->params;
+        $build['form'] = drupal_get_form('media_bynder_add', $params);
```
Comment (danielsequeira) [inline includes/MediaBynderBrowser.inc:29]: why not pass $this->params directly here?

File: includes/media_bynder.formatters.inc [hunk 2]
```diff
@@ -10,7 +10,8 @@
  * Implements hook_file_formatter_info().
  * $info: Array of information on file formatters exposed by hook_file_formatter_info() implementations.
  */
-function media_bynder_file_formatter_info_alter(&$info) {
+function media_bynder_file_formatter_info_alter(&$info)
```
Comment (danielsequeira) [inline includes/media_bynder.formatters.inc:13]: why are some function names camel case and others with underscore?

File: media_bynder.module [hunk 3]
```diff
@@ -118,48 +122,49 @@ function media_bynder_menu() {
...
-  }
-
-  return $libraries;
+function media_bynder_library()
+{
+    $path = drupal_get_path('module', 'media_bynder');
+    $info = system_get_info('module', 'media_bynder');
+
+    $common = array(
+        'website' => 'http://www.getbynder.com',
```
Comment (danielsequeira) [inline media_bynder.module:131]: www.bynder.com

**Review by danielsequeira — CHANGES_REQUESTED** [review]

File: includes/MediaBynderBrowser.inc (hunk 1 above)
Comment (jfcsantos) [reply includes/MediaBynderBrowser.inc:29]: because I didn't touch this function, old code. Done

File: includes/media_bynder.formatters.inc (hunk 2 above)
Comment (jfcsantos) [reply includes/media_bynder.formatters.inc:13]: OOP PHP vs Drupal 7, the camel case ones are normally PHP classes implemented in external libraries and adhering to proper PHP coding standards. Underscore ones are Drupal 7 standards for function names

File: media_bynder.module (hunk 3 above)
Comment (jfcsantos) [reply media_bynder.module:131]: done

**Pull request merged**